cmd/podman/images/scp.go

```diff
@@ -379,3 +426,119 @@ func parseArgs(args []string, cfg *config.Config) (map[string]config.Destination
 	}
 	return serv, nil
 }
+
+// execLoad executes the podman load command given the podman binary
+func execLoad(podman string) error {
+	var loadCommand []string
+	if scpOpts.Save.Quiet || scpOpts.Load.Quiet {
+		loadCommand = []string{"load", "-q", "--input", scpOpts.Save.Output}
+	} else {
+		loadCommand = []string{"load", "--input", scpOpts.Save.Output}
+	}
+	if rootless.IsRootless() {
```

Comment:
I believe you need to ensure that the global options passed to Podman are also passed to the commands you are executing for the local portion of the command.

Reply:
the line `cmd.Args = append(cmd.Args, parentFlags...)` deals with the global options. Is there an easier way to get them?

Reply:
I don't know off hand, but I just wanted to make sure you had taken this into consideration.